packages/enr/src/enr.spec.ts

```diff
@@ -311,6 +314,7 @@ describe("ENR", function () {
     const ip6 = "::1";
     const tcp = 8080;
     const udp = 8080;
+    # TODO: wakuv2 is being decomissioned. What to do with such addresses?
     const wsMultiaddr = multiaddr(
       "/dns4/node-01.do-ams3.wakuv2.prod.statusim.net/tcp/8000/wss"
     );
```

Comment:
we'll have to change the tests to account for the new addr

Reply:
Perhaps this warrants a larger discussion:
Considering these fleets now run The Waku Network, we should refactor not only the tests, but also the docs, to make this the new default
cc @waku-org/js-waku-developers

Reply:
A, it's just a test psec. Let me try.

For example `"/dns4/node-01.do-ams3.wakuv2.prod.statusim.net/tcp/8000/wss"`
will be `"/dns4/node-01.do-ams3.waku.sandbox.statusim.net/tcp/8000/wss"

Reply:
@danisharora099, done. Acn you please review again?

Reply:
> @danisharora099, done. Acn you please review again?

The CI checks seem to be failing. Can you take another look? Let me know if you need any help!

Reply:
@danisharora099, can you please have a look at the 2 issues below.
Looks like I need to do smth with `@waku/dns-discovery` and protobuf.

Reply:
I addressed 
- the `dns-discovery` issue here: https://github.com/waku-org/js-waku/pull/1936
- the protobuf issue here: https://github.com/waku-org/js-waku/pull/1898